### PR TITLE
switch from Adapt.mapById to Adapt.findById

### PR DESCRIPTION
**Pull request by moloko**
see https://github.com/adaptlearning/adapt_framework/issues/2905

also do a bit of a code cleanup

### CODE REVIEW

**Review by olivermartinfoster — APPROVED** [review]

**Review by cahirodoherty-learningpool — APPROVED** [review]
👀

File: js/languagePickerDrawerView.js [hunk 1]
```diff
@@ -46,56 +46,45 @@ define([
...
       this.remove();
 
-      // wait for notify to close fully
-      _.delay(function(){
-        // focus on navigation bar icon
-        this.$finishFocus.a11y_focus();
-      }.bind(this), 500);
+      _.delay(() => {
+        Adapt.a11y.focusFirst(this.$finishFocus);
+      }, 500);
```
Comment (tomgreenfield) [inline js/languagePickerDrawerView.js:87]: One-liner?
```suggestion
      _.delay(() => Adapt.a11y.focusFirst(this.$finishFocus), 500);
```

**Review by tomgreenfield — APPROVED** [review]
:eye:

File: js/languagePickerDrawerView.js (hunk 1 above)
Comment (moloko) [reply js/languagePickerDrawerView.js:87]: possibly... we could do with setting a style guide for these. I know eslint complains if you try to do something like this:
```js
_.defer(() => this._disableAnimationOnce = false);
```
Some people seem to favour only using the one line style when you're returning the value of an expression.

TBH I only left it like that cause it requires minimal effort to change to that style ;-)

Comment (moloko) [reply js/languagePickerDrawerView.js:87]: 😆 fair enough!!

**Review by tomgreenfield — APPROVED** [review]
👁️

**Pull request merged**